test-site/package.json

```diff
@@ -43,4 +43,4 @@
       "last 1 safari version"
     ]
   }
-}
+}
```

Comment:
nit: can you add this newline back. also can you revert the test-site `package-lock` changes